build-test.sh

```diff
@@ -948,6 +948,15 @@ if [[ $__ClangMajorVersion == 0 && $__ClangMinorVersion == 0 ]]; then
     fi
 fi
 
+if [[ "$__BuildArch" == "armel" ]]; then
```

Comment:
Can you hoist this decision out of build-test and put it in init-distro-rid.sh

Reply:
I've updated `build.sh` and `build-test.sh`. Also, I've fixed armel build for priority1 tests.